_startup/biodiversite.md

```diff
@@ -1,8 +1,27 @@
 ---
-title: Plant & Moi
-mission: Améliorer la biodiversité dans nos jardins
-status: investigation
+title: Plante & Moi
+mission: Augmenter la présence de la nature en ville
+status: construction
 contact: celia.henry@developpement-durable.gouv.fr
```

Comment:
not yet, chiant d'ailleurs dans le process : le setup du domaine, avec contact@plante-et-moi.beta.gouv.fr
on pourra y revenir plus tard, pas grave à ce stade sans prod.